banzai/readnoisemap.py

```diff
@@ -0,0 +1,25 @@
+import logging
+
+from banzai.calibrations import CalibrationUser
+from banzai.logs import format_exception
+
+logger = logging.getLogger('banzai')
+
+
+class ReadNoiseMapLoader(CalibrationUser):
+    def apply_master_calibration(self, image, master_calibration_image):
+        try:
+            for image_extension, readnoise_extension in zip(image.ccd_hdus, master_calibration_image.ccd_hdus):
```

Comment:
Is the readnoise map maker prepared to create MEF maps?

Reply:
Not at the moment, but we wanted to make sure that we could accept an MEF map and handle the general case.